services/movie.py

```diff
@@ -23,18 +29,19 @@
 
 
 def create_movie(
-    movie_title: str,
-    movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list = None,
```

Comment:
The type hint `list` for `genres_ids` and `actors_ids` is not specific. Consider using `Optional[List[int]]` for better clarity and compatibility with Python versions earlier than 3.9.